frontend/src/components/SelectApplication.vue

```diff
@@ -6,9 +6,20 @@ import {
     isApplicationSelected,
     selectedApplication,
 } from '@/services/ApplicationState';
-import { onMounted } from 'vue';
+import { onMounted, ref } from 'vue';
 import router from '../router';
 
+const ALL_APPS = 'All';
```

Comment:
Hi Nick, `const environments[0]` is already having value `All`.
Probably no need to define this ALL_APPS const anymore?

Reply:
This const is to make it clear that is for the all_apps option, not array item 0 which might be confusing, specially if we change the app order in the future.